charts/spark-operator-chart/templates/cert-manager-certificate.yaml

```diff
@@ -0,0 +1,29 @@
+{{- $webhookSvc := printf "%s-webhook.%s" (include "spark-operator.fullname" .) .Release.Namespace -}}
+{{- if and .Values.webhook.enable .Values.webhook.certManager.enable }}
+# {{ required "A valid .Values.webhook.certManager.issuerRef is required!" .Values.webhook.certManager.issuerRef }}
```

Comment:
left over?

Reply:
That was a direct copy and paste from our original fork.  I'll remove.